### ISSUE TITLE
Spool Selection Dialog when Spool is already Correct

### ISSUE THREAD

**Issue by GizmoGoody** [malnvenshorn/OctoPrint-FilamentManager#238]
**Is your feature request related to a problem? Please describe.**
It's a small nuisance to have to reselect the already correct spool before starting a print.

**Describe the solution you'd like**
I just started using the "Show dialog to confirm selected spools before starting/resuming the print job" and I mostly like it, except can you change it to not require selecting the same filament from the dropdown if it's already correct? Maybe add a "Continue" button and/or preselect the same spool in the dropdown?

**Describe alternatives you've considered**
Dealing w/ it ;-)

**Additional context**
N/A

<img width="522" alt="Capture" src="https://user-images.githubusercontent.com/46235745/133804202-5b5b44bf-72b2-41bd-ad43-04ececdf31a6.PNG">

**Comment by geeklimit** [comment]
Can confirm this is happening to me as well. "There are no information available about the active tools for the print job" when a valid spool is already selected.  Clicking 'cancel', then restarting the print brings up the dialog box with the current spool selected, as expected, and 'Start print' is able to be clicked. Happens on every print.

**Comment by GizmoGoody** [comment]
> Can confirm this is happening to me as well. "There are no information available about the active tools for the print job" when a valid spool is already selected.  Clicking 'cancel', then restarting the print brings up the dialog box with the current spool selected, as expected, and 'Start print' is able to be clicked. Happens on every print.

I agree this is an issue, but I think it's separate from my request.
